task-list/app/models/person.rb

```diff
@@ -0,0 +1,11 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def self.incomplete(pers_id)
```

Comment:
I like the idea to use a class method here to extend the vocabulary associated with the `Person` model!

As we'll seen soon enough, you can actually move this method into the `Task` class and set `tasks = Task.all` instead of `person.tasks` (no longer any need for a `pers_id` parameter).

This will allow you to view all incomplete tasks with `Task.incomplete` -- however, it will also allow you to view the incomplete tasks for a single person with `person.tasks.incomplete`.